.publishrc

```diff
@@ -5,11 +5,11 @@
     "untrackedFiles": true,
     "sensitiveData": true,
     "branch": "master",
-    "gitTag": true
+    "gitTag": false
   },
   "confirm": true,
   "publishCommand": "npm publish",
-  "publishTag": "beta",
+  "publishTag": "next",
```

Comment:
`Check that git tag matches version specified in the package.json`

Wouldn't we still want this?

Reply:
Just following what is in the CONTRIBUTING.md.  `next` for .publishrc and `-dev1` in the version property.